CONTRIBUTING.md

```diff
@@ -59,6 +59,32 @@ Output format of any non-plumbing command can change in non-backward-compatible
 * [Nifty Docker tricks for your CI (vol. 2)](https://medium.com/virtuslab/nifty-docker-tricks-for-your-ci-vol-2-c5191a67f1a4)
 
 
+## FAQ's about Pull Requests
+
+**Who closes Github comments?** 
+
+Author of changes, reviewer or initiator of the conversation. It makes sense to close comment:
```

Comment:
```suggestion
**Who closes Github comments? Author of changes, reviewer or initiator of the conversation?**

It makes sense to close the comment:
```
(since that second sentence was a part of the question, not the answer)

Reply:
Also! pls fix the failing CI check (trailing whitespace in line 64 & 67)

Reply:
(Select Log in via Github in CircleCI btw if needed)